corehq/apps/data_interfaces/interfaces.py

```diff
@@ -183,12 +180,17 @@ def template_context(self):
         context.update({
             "form_query_string": self.request.GET.urlencode(),
             "mode": self.mode,
-            "total_xForms": int(self.es_results['hits']['total']),
+            "total_xForms": int(self.es_query_result.total),
         })
         return context
 
-    def _es_xform_filter(self):
-        return self.mode.xform_filter
+    @property
+    def es_query(self):
+        query = super(BulkFormManagementInterface, self).es_query
+        if self.mode.mode_name == self.mode.RESTORE_MODE:
+            return query.only_archived()
+        else:
+            return query
```

Comment:
No, archived forms are excluded by default from all FormES queries, since they have a different doctype.  The `only_archived()` method removes that default filter and adds a filter to exclude non-archived forms.